types/roundtrip.go

```diff
@@ -42,6 +42,10 @@ type Response struct {
 	// StatusCode checks response code
 	StatusCode int `json:"statusCode"`
 
+	// Headers defines http header of request
+	// NOTE(liubog2008): whether to use map[string][]string
```

Comment:
do you mean we should try using `map[string][]string` here?

Reply:
yes